features/polyExplorer/src/components/entityList/entityList.jsx

```diff
@@ -58,19 +17,26 @@ function getStartGroups(entityGroups) {
     return keys.pop();
 }
 
-const EntityList = () => {
-    const { entities, globalData, activeFilters, handleRemoveFilter } =
-        useContext(ExplorerContext);
-    const onRemoveFilter = handleRemoveFilter;
-    const filteredEntities = activeFilters.apply(entities);
-    const entityGroups = groupEntities(filteredEntities);
+export default function EntityList({ entityGroups }) {
     const allKeys = Object.keys(entityGroups);
     const [loadedEntities, setLoadedEntities] = useState({});
     const [toLoadKeys, setToLoadKeys] = useState(allKeys);
```

Comment:
(off-scope) naming nit: maybe keysToLoad?

Reply:
Ended up being `groupsToLoad`, cause that's what it is :P